pkg-manager/core/test/install/fixLockfile.ts

```diff
@@ -233,3 +233,94 @@ test('--fix-lockfile should preserve all locked dependencies version', async ()
   })
   expect(lockfile.packages?.['/regenerator-runtime/0.13.9']?.dev).toBeFalsy()
 })
+
+test(
+  '--fix-lockfile should install successfully when package has no dependencies but has peer dependencies with version like /vite/4.0.4_@types+node@17.0.45',
+  async () => {
+    preparePackages([
+      {
+        location: '.',
+        package: { name: 'root' },
+      },
+      {
+        location: 'project-1',
+        package: { name: 'project-1', devDependencies: { vite: '4.0.4' } },
+      },
+      {
+        location: 'project-2',
+        package: { name: 'project-2', devDependencies: { vite: '4.0.4', '@types/node': '17.0.45', '@vitejs/plugin-basic-ssl': '1.0.1' } },
```

Comment:
We have our own lightweight packages for testing peer dependencies and other stuff. Can the test be rewritten to reuse our packages? 

I believe, instead of `@vitejs/plugin-basic-ssl`, you may use this package: https://github.com/pnpm/registry-mock/blob/main/packages/has-foo100-peer/package.json

Also, should the test be testing a workspace? Isn't it enough to just test on a single project?

Reply:
Thanks. I have updated the test case and don't use workspace.